frontend/smart-kickers-game/src/components/GameResults.css

```
@@ -0,0 +1,19 @@
+.game-result-container {
+  padding-top: 3rem;
+  display: flex;
+  justify-content: center;
+  font-size: 3rem;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.game-result-item {
+  padding: 0 3rem 0 3rem;
```

```suggestion
  padding: 0 3rem;
```
Just a quick info that you can simplify this notation.